cmd/agent/app/agent.go

```diff
@@ -205,7 +207,7 @@ func run(ctx context.Context, opts *options.Options) error {
 		LeaseDuration:              &opts.LeaderElection.LeaseDuration.Duration,
 		RenewDeadline:              &opts.LeaderElection.RenewDeadline.Duration,
 		RetryPeriod:                &opts.LeaderElection.RetryPeriod.Duration,
-		HealthProbeBindAddress:     net.JoinHostPort(opts.BindAddress, strconv.Itoa(opts.SecurePort)),
```

Comment:
https://github.com/karmada-io/karmada/blob/master/artifacts/agent/karmada-agent.yaml#L34

Should we also make some changes here?

Reply:
> Should we also make some changes here?

Yes, considering that the old flags will be removed when upgrading from `1.11` to `1.12`, replacing the old flags with the new ones at `1.11` is a good choice